.github/workflows/build-images-action.yml

```diff
@@ -0,0 +1,30 @@
+name: build-images-action
+on:
+  push:
+    branches:
+      - main
+      - 'release-*'
+    tags:
+      - 'v**'
```

Comment:
Whats the syntax here? Regex, glob? 

This also matches `apis/v....` so probably needs start of line symbol.

Reply:
It's said to use glob:
https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#using-filters-to-target-specific-branches-for-pull-request-events

Reply:
Thanks for the clarification. I've an impression that I saw v*/* pattern somewhere, but if that's not what we want, then `v*` should suit better.